### ISSUE TITLE
About IJB validation result of model trained with Glint360k 

### ISSUE THREAD

**Issue by overpresentme** [deepinsight/insightface#1292]
After training IResnet-100 with Glint360k, I did validation of IJB-B and IJB-C.

TPR@FPR=1e-4 of IJB-C was 97.10% (closed to your Partial-FC paper result = 97.30%). 

But TPR@FPR=1e-6 of IJB-C was too low(82%) compared to other models(92.0%) trained with different datasets (ex: MS1MV2)

You know, the result of TPR(@FPR=1e-4 of IJB-B & IJB-C) has been mainly addressed in recent papers.

What do you think of which result is more relevant in "NIST FRVT 1:1 rank" between TPR@FPR=1e-4 and TPR@FPR=1e-6?

Could you share your opinion about this?

Thanks for reading.

**Comment by nttstar** [comment]
How many epochs ?

**Comment by mlourencoeb** [comment]
@overpresentme what loss have you used? CosFace, Arcface?

**Comment by overpresentme** [comment]
My Train Configuration:
- Loss: ArcFace (s=64.0, m=0.5)
- batch size is 8 x 64 (512)
- first learning rate is 0.1 and divided by 10 in (5.85 / 11.7 / 14.6 / 16 epoch). training is ended in 31 epoch.

**Comment by nttstar** [comment]
Try to train at least 10 epoches on LR=0.1.
And IJBC e-4 and e-5 are more discriminative.
Or you can submit your models to test on our [IFRT](https://github.com/deepinsight/insightface/tree/master/IFRT).

**Comment by overpresentme** [comment]
@nttstar Thanks for your advice!

**Comment by xiaoshudi** [comment]
@nttstar 
After training Resnet-50 with VGGface2, I did validation of IJB-B.

TPR@FPR=1e-4 of IJB-B was 59.29% (far away from your paper result = 89.80%).

Loss: ArcFace (s=64.0, m=0.5)
batch size is 512
first learning rate is 0.1 and divided by 10 in (7 / 13 epoch). training is ended in 28 epoch.

![2020-12-07 11-08-07屏幕截图](https://user-images.githubusercontent.com/38290717/101305256-3c612900-387d-11eb-88a8-c84410bf0d1d.png)

Could you share your opinion about this?

Thanks for reading.
